cli/standalone.rs

```diff
@@ -219,6 +223,79 @@ fn metadata_to_flags(metadata: &Metadata) -> Flags {
   }
 }
 
+fn web_worker_callback() -> Arc<WorkerEventCb> {
+  Arc::new(|worker| {
+    let fut = async move { Ok(worker) };
+    LocalFutureObj::new(Box::new(fut))
+  })
+}
+
+fn create_web_worker_callback(
+  ps: &ProcState,
+  module_loader: &Rc<EmbeddedModuleLoader>,
+) -> Arc<CreateWebWorkerCb> {
+  let ps = ps.clone();
+  let module_loader = (module_loader as &EmbeddedModuleLoader).clone();
```

Comment:
`(module loader as &EmbedderModuleLoader)` is a deref without needing to `use std::ops::Deref`. I can use that instead if it makes it clearer.

`module_loader.clone()` would clone the `Rc`, which doesn't work because the `CreateWebWorkerCb` callback must be `Send`.

Looking at this again, it seems like the `EmbeddedModuleLoader` is never mutated after creation, and the eszip is already put behind an `Arc`. So I guess I could also put the `maybe_import_map_resolver` behind an arc, so cloning an `EmbeddedModuleLoader` is cheap.

Reply:
I... Understood some of that :D

Any way you feel is good, my knowledge isn't good enough to judge if an `Arc` would make more sense here or if the current double-clone way makes more sense.